### ISSUE TITLE
Enemy AI squad gets stuck after being spawned

### ISSUE THREAD

**Issue by fugasjunior** [KillahPotatoes/KP-Liberation#886]
### Basic Information

**Mission version:** 0.96.7a

**Map used:** Altis

**Mods used:** none

**Dedicated or local game:** dedicated server, 64bit, Linux

**Source of the mission file (URL):** https://steamcommunity.com/sharedfiles/filedetails/?id=878714108

### Individual Things
**Did you edit anything within the mission files?**
No

### Short Issue Description
When an enemy squad spawns, the squad leader gets stuck among the other units a lot of the time and cannot move to any of the given waypoints. This causes the whole squad to be stuck in place and not patrolling the area.

This can be manually fixed via the Zeus interface by moving the squad leader and any other units that are potentially stuck a few meters away so that they can follow their waypoints.

This happens both with several mods enabled (incl. ACE, Enhanced movement, ...) and with no mods whatsoever.

### Expected Behaviour

None of the squad members gets stuck and the whole squad starts moving according to their waypoints.

### Steps to Reproduce the Issue

1. Spawn as a Platoon commander and open Zeus interface
2. Wait until enemy units spawn (personally observed in the area of Alikampos)
3. Observe spawned squads of 8 people being stuck in place due to leader being blocked by other units

### Screenshots, Serverlog or any other helpful Information
![unknown](https://user-images.githubusercontent.com/17858781/123508092-853bcc00-d66d-11eb-88df-0a6a7f8e520a.png)

**Comment by xZToxicZx** [comment]
I'm having the same issue however even moving the squad leader doesn't fix it

**Comment by MildlyInterested** [comment]
Are you running Advanced Rappelling or any of the other Advanced Mods on your server?
Remove them and try again.

**Comment by FaustKatt** [comment]
This is happening to friendly AI squads I buy as well (no mods but CBA). It seems to be related to their default formation being set to column; it's really hard for them to unwind and get going because the later ones in the column don't move until the early ones do.

**Comment by nicoman35** [comment]
I think this is just another 'AI getting stuck' problem. Vehicles get stuck on objects or terrain, infantry units get stuck in buildings , rocks or sometimes between other infatry units, if there are lots of stationary infatry units around them. Extremely annoying. Playing as a single player with lots of own ai units reveals this problem. They respond to move orders 'CANT GET THERE'.
Solution to this is only possible with some ugly stuck detection loop. But Arma AI is to blame for that in first place. Should be devs job to fix this. Not ours.

**Comment by Kogs** [comment]
This happens all the time for me. Are there any fixes? It ruins all the fun. 
Do they just need more time to get going or what? 
All the ai is just standing there. And if they start shooting they don't even raise their weapons.

**Comment by MildlyInterested** [comment]
Don't run advanced rappelling or the other advanced something mods from the same modmaker. They're causing this, why idk.

**Comment by Kogs** [comment]
> Don't run advanced rappelling or the other advanced something mods from the same modmaker. They're causing this, why idk.

Yeah I figured, after removing enhanced movement etc from the server the ai is working fine again.
